test/src/test/java/org/corfudb/runtime/view/StreamViewTest.java

```diff
@@ -29,16 +29,76 @@
  */
 public class StreamViewTest extends AbstractViewTest {
 
-    @Getter
-    final String defaultConfigurationString = getDefaultEndpoint();
-
     public CorfuRuntime r;
 
     @Before
     public void setRuntime() throws Exception {
         r = getDefaultRuntime().connect();
     }
 
+    @Test
+    public void nonCacheableStream() {
+        UUID id1 = UUID.randomUUID();
+        UUID id2 = UUID.randomUUID();
+
+        CorfuRuntime producer = getNewRuntime(getDefaultNode())
```

Comment:
Can we surround in a try/catch/finally and shutdown the producer and consumer at the end?

Reply:
Its cleaned by `AbstractViewTest::cleanupBuffers`